serve.go

```diff
@@ -203,11 +203,13 @@ func serve(args []string) error {
 
 	pushHandlers := nanopush.MakeHTTPHandlers(ctx, pushEndpoints, checkinOpts...)
 	scepHandler := scep.ServiceHandler(ctx, sm.scepService, httpLogger)
-	enrollHandler := enroll.ServiceHandler(ctx, sm.enrollService, httpLogger)
+	enrollHandlers := enroll.MakeHTTPHandlers(ctx, enroll.MakeServerEndpoints(sm.enrollService, sm.scepDepot), httptransport.ServerErrorLogger(httpLogger))
 	r := mux.NewRouter()
 	r.Handle("/mdm/checkin", mdmAuthSignMessageMiddleware(sm.scepDepot, checkinHandlers.CheckinHandler)).Methods("PUT")
 	r.Handle("/mdm/connect", mdmAuthSignMessageMiddleware(sm.scepDepot, connectHandlers.ConnectHandler)).Methods("PUT")
-	r.Handle("/mdm/enroll", enrollHandler).Methods("GET", "POST")
+	r.Handle("/mdm/enroll", enrollHandlers.EnrollHandler).Methods("GET", "POST")
+	r.Handle("/ota/enroll", enrollHandlers.OTAEnrollHandler)
+	r.Handle("/ota/phase23", enrollHandlers.OTAPhase2Phase3Handler).Methods("POST")
```

Comment:
Can you think of a more descriptive name than `phase23`? Maybe we can name this `/ota/finalize` ? or `/ota/start` and `/ota/complete`?